clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -187,10 +191,16 @@ export class CopybookDownloadService implements vscode.Disposable {
                     throw e;
                 }
             }
-            vscode.window.showErrorMessage(errorMessage);
+            if (CopybookDownloadService.isQueueNotQuiet(toDownload)) {
+                vscode.window.showErrorMessage(errorMessage);
+            }
         }
     }
 
+    private static isQueueNotQuiet(queue: CopybookProfile[]): boolean {
```

Comment:
I would prefer avoiding negations in boolean methods to keep the code more clear. 
From the API point of view, it may be something like `needsUserNotification`

Reply:
Agree, thanks.